### PR TITLE
Readds poly instead of Bob

### PR DESCRIPTION
**Pull request by dragoslava**

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -681,7 +681,7 @@
 /mob/living/simple_animal/parrot/Poly
 	name = "Poly"
 	desc = "Poly the Parrot. An expert on quantum cracker theory."
-	speak = list("Poly wanna cracker!", ":e Check the singlo, you chucklefucks!",":e Wire the solars, you lazy bums!",":e WHO TOOK THE DAMN VOIDSUITS?",":e OH GOD ITS FREE CALL THE SHUTTLE")
+	speak = list("Poly wanna cracker!", ":e What do you call a sketchy italian neighbourhood? THE SPAGHETTO SQAWWW SQAAWWW!!",":e Setup the engine, you lazy bums!","Kep't you waiting huh? Kept you waiting huh?","Venus in cancer....")
```
Comment (Alberyk) [inline code/modules/mob/living/simple_animal/parrot.dm]: No, it makes it worse, by replacing jokes that have to do with the game culture with puns and mgs references. Either don't touch those or just remove the jokes, if those are an issue.

**Comment by Arrow768** [comment]
DNM until after the update due to map-freeze for new changes 1-week prior.

**Comment by skull132** [comment]
Poly's going to be contentious. Because it's literally an automated griefer.

**Comment by LordFowl** [comment]
BOB did nothing wrong. All poly does is shitpost and steal.

**Comment by OneOneThreeEight** [comment]
Mildly curious: How is Poly an automated griefer? I've never heard that described of Poly before.

**Comment by Arrow768** [comment]
Merge conflict has not been fixed. Moving to dezember

**Pull request closed without merging**